ew/cmd/spooky/spookyutils.py

```diff
@@ -24,3 +24,20 @@ def generate_negaslimeoid_name():
     name = name.capitalize()
     full_name = "{}, {}".format(name, title)
     return full_name
+
+class EwChef:
```

Comment:
this needs to be moved inside the model directory, either by using an existing file there or creating a spooky.py file